vaapi/vaapisurfaceallocator.h

```diff
@@ -24,12 +24,16 @@ namespace YamiMediaCodec{
 
 class VaapiSurfaceAllocator : public BaseSurfaceAllocator
 {
+public:
     //extra buffer size for performance
     static const uint32_t EXTRA_BUFFER_SIZE = 5;
+    static const uint32_t MINIMUM_EXTRA_BUFFER_SIZE = 2;
```

Comment:
as I mentioned, this not needed. Only thing we need is let user set the extras buffer size

Reply:
If so, user can not know how many surfaces are created.